src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -29,7 +29,7 @@ internal sealed class PowerShellConfig
 
         private string psHomeConfigDirectory;
         private string appDataConfigDirectory;
-        private const string configFileName = "PowerShellProperties.json";
+        private const string configFileName = "powershell.config.json";
```

Comment:
about_Logging.md will need to be updated. 
See https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Logging.md

Reply:
Opened https://github.com/PowerShell/PowerShell-Docs/issues/2017 to track the documentation update.

Reply:
Note that about_logging.md needs to be updated regardless since it refers to 'PowerShellProperties.json'.